lib/text/html-lite.scm

```diff
@@ -165,10 +165,6 @@
 
 (define-html-elements body address div)
 
-;; SEMANTIC ELEMENT
```

Comment:
This is the "semantic element" group, should we keep it (and maybe add more)? And have a separate group for html5 non-semantics elements?

Reply:
No idea about what the "SEMANTIC ELEMENT" means.
As far as I can tell, neither HTML 5.2 specification (nor previous versions of HTML5) defines such a group of elements.
In fact the document says that (all) "Elements, attributes, and attribute values in HTML are defined (by this specification) to have certain meanings (semantics)." [1]
Found only some informal pages talking about the distinction of "semantic" ones from others when searching "html semantic non-semantic elements" via google.

[1] https://www.w3.org/TR/html5/dom.html#elements-semantics

Reply:
Never mind. I looked back in history and this group _is_ html5 (in d73d2c1f9903de924c922bb4fd2e0708e9949724). So yeah looking good. Sorry for the noise.